.github/workflows/release.yaml

```
@@ -10,20 +10,26 @@ jobs:
   check:
     uses: ./.github/workflows/check.yaml
     secrets: inherit
-
   release:
     needs: check
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.runs-on }}
+    strategy:
+      fail-fast: false
+      matrix:
+        runs-on: [[ubuntu-latest], [Ubuntu_ARM64_4C_16G_01]]
```

not-blocking: It's just more readable.
```suggestion
        runs-on: 
          - [ubuntu-latest]
          - [Ubuntu_ARM64_4C_16G_01]
```

---

Done